cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionFilter.java

```diff
@@ -141,6 +141,32 @@ public PartitionFilter build() {
       Preconditions.checkState(!map.isEmpty(), "Partition filter cannot be empty.");
       return new PartitionFilter(map);
     }
+
+    /**
+     * @return <tt>true</tt> if the underlying map contains no key-value mappings
```

Comment:
I actually needed this in a separate branch. Removing it from here and will update the javadoc in that branch.